Essentials/src/messages.properties

```diff
@@ -377,6 +377,7 @@ onlySunStorm=\u00a74/weather only supports sun/storm.
 openingDisposal=\u00a76Opening disposal menu...
 orderBalances=\u00a76Ordering balances of\u00a7c {0} \u00a76users, please wait...
 oversizedTempban=\u00a74You may not ban a player for this period of time.
+passengerTeleportFail=\u00a74Teleporting requires no passengers and teleport-passenger-dismount is disabled.
```

Comment:
This doesn't seem to be a very player-friendly message. It seems to be geared more towards server admins. I think the description of the setting should be solely in the config. Maybe change this to something like "You cannot teleport while carrying passengers." or something like that.